packages/mobile/src/firebase/saga.ts

```diff
@@ -142,4 +157,6 @@ export function* firebaseSaga() {
   yield spawn(initializeFirebase)
   yield spawn(watchLanguage)
   yield spawn(subscribeToCeloGoldExchangeRateHistory)
+  yield takeLatest(AppActions.APP_MOUNTED, watchFirebaseNotificationChannel)
```

Comment:
`takeLatest` is not blocking, right? I ask because we do the `spawn` thing everywhere else, but if it's not blocking this looks fine :)

Reply:
Good question. The reason we wanted takeLatest is as per Jean's suggestion to not have to manually cancel. Here is the implementation: 

`const takeLatest = (patternOrChannel, saga, ...args) => fork(function*() {
  let lastTask
  while (true) {
    const action = yield take(patternOrChannel)
    if (lastTask) {
      yield cancel(lastTask) // cancel is no-op if the task has already terminated
    }
    lastTask = yield fork(saga, ...args.concat(action))
  }
})`